ci/packaging/suse/obsfiles_maker.sh

```diff
@@ -19,14 +17,16 @@ trap clean ERR
 rm -rf "${obs_files}"
 mkdir -p "${obs_files}"
 
-cp "${image}/${image}.kiwi" "${tmp_dir}"
-[ -f "${image}/config.sh"  ] && cp "${image}/config.sh" "${tmp_dir}"
-[ -d "${image}/root"  ] && tar -caf "${tmp_dir}/root.tar.gz" \
-    -C "${image}/root" .
-[ -f "${image}/_service"  ] && cp "${image}/_service" "${tmp_dir}"
-make CHANGES="${tmp_dir}/${image}.changes.append" suse-changelog
+install -p -m 644 "${image}/${image}.kiwi" "${obs_files}"
+[ -f "${image}/config.sh"  ] && install -m 644 -p "${image}/config.sh" "${obs_files}"
+[ -f "${image}/_service"  ] && install -m 644 -p "${image}/_service" "${obs_files}"
+
+[ -d "${image}/root"  ] && \
+    tar --mtime=@1480000000 --owner=0 --group=0 --no-acls --no-xattrs \
+    --pax-option=exthdr.name=%d/PaxHeaders/%f,atime:=0,ctime:=0  \
+    --mode=a+X,u+r -caf "${obs_files}/root.tar.gz" -C "${image}/root" .
 
-cp "${tmp_dir}"/* "${obs_files}"
-log "Find files for RPM package in ${obs_files}"
-clean
 
+make CHANGES="${obs_files}/${image}.changes.append" suse-changelog
```

Comment:
This assumes the cwd is $image (for the makefile), which could potentially not be the case depending on how the script was run.  Seems like it might make more sense to edit the Makefile to include suse-changelog when suse-package target is built. Calling make in a script called from make causes me some recursive fear either way. :)

Reply:
yeah, but I didn't change that or introduce that. I only moved that line of text (and removed the tmpdir stuff). 

agree to do a followup on that?I don't want to introduce too many cleanups in one PR

Reply:
Ah, I missed the old line (I blame the mobile view).

Yes, separate PR would be better. :D

Reply:
SUSE/avant-garde#1872